### ISSUE TITLE
openbsd-file doesn't work as root (doesn't create unprivileged user on install)

### ISSUE THREAD

**Issue by loreb** [void-linux/void-packages#19102]
### System

* xuname:  
  Void 5.4.19_1 x86_64-musl GenuineIntel uptodate rFFF

* package:  
  openbsd-file-6.2_2

### Expected behavior
Can be safely run as root (which is part of the reason this implementation exists after all)

### Actual behavior
```
# file *; echo exited $?
openbsd-file: unknown user _file
exited 1
```

### Steps to reproduce the behavior
just remember to run as root

**Comment by ailiop-git** [comment]
#19203

**Comment by xtraeme** [comment]
fixed via #19203
